gems/smithy/lib/smithy/vise/enum_shape.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Vise
+    # Represents an enum shape from the model.
+    class EnumShape < Shape
```

Comment:
General thought / style question - is it worth having tons of these small shape files or, as we do in V3, should it just be a single file?

Reply:
I don't think it matters too much. I can put them in one file if that helps.

Reply:
I think the file would be too big otherwise. These shape classes do more than the runtime ones.